javascript/react_js/router.md

```diff
@@ -103,7 +103,7 @@ You should now have enough basics to get started with React routing. There are a
 <div class="lesson-content__panel" markdown="1">
 1. Go and add a few new routes to the application we created above; playing around with it is the best practice. Consider deleting it completely and rewriting it for practice.
 2. Browse a little bit through the React Router [documentation](https://reactrouter.com/en/main/start/overview). You don't need to read through all of it, nor understand all of it. Just browse through the concepts we discussed here and re-read them. This is a great resource to refer back to.
-3. Watch this [video on React Router by Dev Ed](https://www.youtube.com/watch?v=Law7wfdg_ls) for reviewing. You should already be familiar with those concepts.
+3. Watch this [video on React Router by Dev Ed]([https://www.youtube.com/watch?v=Law7wfdg_ls](https://www.youtube.com/watch?v=Ul3y1LXxzdU)) for reviewing. You should already be familiar with those concepts.
```

Comment:
The author of the video changed, as well as your change included 2 different urls. You want to make sure that the link description is inside `[]` and the url is inside `()`, like: `[link description](url)`

```suggestion
3. Watch this [video on React Router by Web Dev Simplified](https://www.youtube.com/watch?v=Ul3y1LXxzdU) for reviewing. You should already be familiar with those concepts.
```

Reply:
Thanks for explaining the steps. I was not sure of the use of [ ] and ( ).

Reply:
@SamAmann As a heads up, the person that requested changes should be the one to `Resolve conversation`, so that they can check if the changes actually resolve it before approving the PR.

There are no additional commits on this PR, so these changes has not been implemented. You can either click on `Commit suggestion` above or make another commit with any changes that you'd like to make.